### DISCUSSION TITLE
Add keyboard shortcuts to rename folders & files

### DISCUSSION THREAD

**Discussion by arthurvanl** [Kong/insomnia#7905] [Ideas]
In postman you could rename a file with `Ctrl + E`. I am very used to this but now it justs opens the environment. Which is also useful. Would like to see a shortcut for renaming a file or folder in insomnia. It also very annoying that when I add a new request it defaultly says `New Request`. In postman the cursor is selected in this text field so you can update the name right away.

**Comment by arthurvanl** [comment]
bump

**Comment by arthurvanl** [comment]
bump

**Comment by VewDev** [comment]
bump. It should be kinda easy to be able to do this from a keybind. All other options in the request dropdown but "Copy as cURL" have a keybind. For me I'd like F2 since I am used to VS Code and the OS file explorer, but I guess any keybind would be fairly easy to learn (or just change to F2 in settings).

**Comment by Alirexaa** [comment]
I'd like to rename requests with F2 too.
![insomnia](https://github.com/user-attachments/assets/94694da8-769b-4475-8b98-0f81d4464a9a)

`F2` is appropriate because it's the default key for renaming in most IDEs and OS file explorer.

**Comment by arthurvanl** [comment]
![image](https://github.com/user-attachments/assets/8ac78cc9-88c4-43a5-b62c-ad5a57a20590)
I dont have a short cut. In the Preferences -> Keyboard, I dont see the shortcut

**Reply by VewDev** [reply]
The previous image is edited. There is no shortcut there as of right now.
